# Supereight ROS node settings
enable_tracking: false

enable_rendering: true

input_size: [640, 480]



# Supereight settings
compute_size_ratio: 2

tracking_rate: 1

integration_rate: 3

rendering_rate: 4

volume_resolution: [256, 256, 256]

volume_size: [24.0, 24.0, 24.0]

initial_pos_factor: [0.25, 0.25, 0.25]

dump_volume_file: ""

T_BC: [0.0, 0.0, 1.0, 0.0, -1.0, 0.0, 0.0, 0.0, 0.0, -1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]

camera: [570.3422241210938, 570.3422241210938, 314.5, 235.5]

camera_overrideds: false

mu: 0.1

fps: 0

blocking_read: false

icp_threshold: 0.00001

no_gui: false

render_volume_fullsize: false

bilateral_filter: false

coloured_voxels: false

multi_resolution: false

bayesian: false

